dcos/src/main/scala/io/vamp/container_driver/marathon/ReadApps.scala

```diff
@@ -10,14 +10,14 @@ case class App(
   id:           String,
   instances:    Int,
   cpus:         Double,
-  container:    Option[AppContainer],
   mem:          Double,
-  env:          Map[String, String],
-  cmd:          Option[String],
-  tasks:        List[Task],
-  healthChecks: List[MarathonHealthCheck],
-  taskStats:    Option[MarathonTaskStats],
-  networks:     List[AppNetwork]
+  container:    Option[AppContainer]      = None,
+  env:          Map[String, String]       = Map(),
+  cmd:          Option[String]            = None,
+  tasks:        List[Task]                = Nil,
```

Comment:
In some cases you initialised into object, and other into Nil.
I think initialising into the empty object would be more safer, like you did in env.